src/debrid-collector/Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
 
-RUN apk add --update --no-cache python3=~3.11.9-r0 py3-pip && ln -sf python3 /usr/bin/python
+RUN apk add --update --no-cache python3 py3-pip && ln -sf python3 /usr/bin/python
```

Comment:
We still require python version 3.11, at a max due to the dependencies we use.
However - you can pin like this instead of using the patch version

```bash
RUN apk add --update --no-cache \
    python3=~3.11 \
    py3-pip=~23.3 \
    curl=~8.9 \
    && ln -sf python3 /usr/bin/python
```

Reply:
Done.